twilight-validate/src/command.rs

```diff
@@ -535,6 +584,67 @@ fn name_characters(value: impl AsRef<str>) -> Result<(), CommandValidationError>
     Ok(())
 }
 
+/// Validate a single name localization in a [`CommandOptionChoice`].
```

Comment:
A single name localization map/dictionary. Still feels a bit off to use the "a single" terminology, but let's keep it if it's the standard.